client/src/components/CardDialog/DialogMenuButton/DialogMenuButton.tsx

```diff
@@ -0,0 +1,81 @@
+import { useState, MouseEvent } from 'react';
+import { Button, Menu, MenuItem } from '@material-ui/core';
+import { useSnackBar, useKanban } from '../../../context';
+import dialogActionButtonStyles from '../DialogActionButton/dialogActionButtonStyles';
+
+type Props = {
+  name: string;
+};
+
+const DialogMenuButton = ({ name }: Props): JSX.Element => {
+  const classes = dialogActionButtonStyles();
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const { updateSnackBarMessage } = useSnackBar();
+  const { activeBoard, moveCard, copyCard, removeActiveCard, getColumnById, focusedCard } = useKanban();
+
+  const handleClick = (event: MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+    updateSnackBarMessage(`Choose a column to ${name.toLowerCase()} current card to`);
+  };
+
+  const handleMenuClose = (target: string) => {
+    setAnchorEl(null);
+
+    // checks to make sure target is actually a column name, then formats text for snackbar
+    // depending on if the action ends in 'y' or not
+    !target.toString().startsWith('[') &&
+      updateSnackBarMessage(
+        `${name.charAt(name.length - 1) === 'y' ? name : name.slice(0, -1)}ing to column: "${
+          getColumnById(target).name
+        }"`,
+      );
+
+    switch (name) {
+      case 'Move':
+        if (getColumnById(target) !== null) {
+          moveCard(getColumnById(target));
+        }
+        break;
+      case 'Copy':
+        if (getColumnById(target) !== null) {
+          copyCard(getColumnById(target));
+        }
+        break;
+      case 'Delete':
+        updateSnackBarMessage(`Deleting card: "${focusedCard?.name}"`);
+        removeActiveCard();
+        break;
```

Comment:
This is good @street-jackal, however, when you delete a card that card is no longer useful so maybe we should close the card dialog after that action is complete.

Reply:
There should be a method for this on the Kanban Context called `resetOpenCard` which will close the actively focused card.

Reply:
I meant to implement this! I'll add that for sure.

Reply:
Ok, now I'm closing the dialog for Delete/Move, but leaving it open for Copy since conceivably a user could want to copy the active card to multiple places at once

Reply:
For sure, it would be cool in the future we actually made sure that when they copy a card the copied card is focused say if they want to add a due date on the copied card.